clients/admin-ui/src/features/system/ManualSystemFlow.tsx

```diff
@@ -88,17 +97,8 @@ const ManualSystemFlow = () => {
       content: activeSystem ? (
         <ReviewSystemStep
           system={activeSystem as System}
-          onSuccess={() => setCurrentStepIndex(currentStepIndex + 1)}
-        />
-      ) : null,
-      isDisabled: !activeSystem,
-    },
-    {
-      label: STEPS[3],
-      content: activeSystem ? (
-        <SystemRegisterSuccess
```

Comment:
it didn't make sense to me to have a success tab (to hit this, you have to enable nav v1, then try editing a system)